<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textBrowse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>接收数据</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendBt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>368</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="IPEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>192.168.90.62</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PortEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>50001</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectBt">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>接入服务器</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
